content/docs/command-reference/metrics/index.md

```diff
@@ -46,6 +46,23 @@ $ dvc metrics diff
 summary.json   AUC      0.801807   0.037826
 ```
 
+`dvc metrics` subcommands by default use the metric files specified in
+`dvc.yaml` (if any), for example `summary.json` below:
+
+```yaml
+stages:
+  train:
+    ...
```

Comment:
I think it's fine to put outs and deps here - to give more context

Reply:
```
    deps:
      - users.csv
```
? Not sure how that gives more context for this metrics example. I added the full YAML file again instead. See 2c506767